app/src/components/v-dialog/v-dialog.vue

```diff
@@ -2,21 +2,17 @@
 	<div class="v-dialog">
 		<slot name="activator" v-bind="{ on: () => (internalActive = true) }" />
 
-		<teleport v-if="placement === 'right'" to="#drawer-outlet">
+		<teleport to="#dialog-outlet">
 			<transition-dialog @after-leave="leave">
-				<div v-if="internalActive" class="container" :class="[className, placement]">
+				<component
+					:is="placement === 'right' ? 'div' : 'span'"
```

Comment:
Smart! Using a different tag so that the `:nth-of-type` trick in CSS still works ❤️